utils/run_test.sh

```diff
@@ -54,6 +54,26 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES=(${DAOS_BASE}/test_results/*.xml)
+    for file in "${FILES[@]}"
+    do
+        if [ -f "$file" ]; then
+            echo "Processing XML $file"
+
+            SUITE=$(grep "testsuite name=" "$file" | \
+                   grep -Po "name=\"\K.*(?=\" time=)" || true)
+
+            CLASS=$(grep "<testcase classname" "$file" || true)
+            if [ "$CLASS" == "" ]; then
+                sed -i \
+                "s/case name/case classname=\"${COMP}.${SUITE}\" name/" "$file"
+            else
+                sed -i "s/case classname=\"/case classname=\"${COMP}./" "$file"
+            fi
```

Comment:
Same issues here as in src/tests/ftest/scripts/main.sh.  This is duplicated code in fact, that really should be DRYed out.

Somebody is going to end up fixing/enhancing something in one of them and not the other.